request_a_govuk_domain/request/forms.py

```diff
@@ -149,6 +149,16 @@ def __init__(self, *args, **kwargs):
         self.helper.layout = Layout(
             Fieldset(
                 Field.text("domain_name"),
+                DomainsHTML(
```

Comment:
I think DomainsHTML.warning method should work here?

Reply:
Thanks. I'd forgotten about those.